src/search/search.go

```diff
@@ -37,6 +39,20 @@ func PerformSearch(query string, maxPages int, visitPages bool) []structures.Res
 		}
 	})
 
+	worker.Go(func() {
+		err := duckduckgo.Search(context.Background(), query, &relay, &options)
+		if err != nil {
+			log.Error().Err(err).Msg("Failed searching lite.duckduckgo.com")
+		}
+	})
+
+	worker.Go(func() {
+		err := mojeek.Search(context.Background(), query, &relay, &options)
+		if err != nil {
+			log.Error().Err(err).Msg("Failed searching mojeek.com")
```

Comment:
Maybe make the link or name of the SE public and use it here in case it ever changes? So that it doesn't happen that we miss a spot here and forget to change it.

Reply:
Agreed